docs/guides/kernel.md

```diff
@@ -41,8 +41,19 @@ done
 !!! Info "Using config from lower kernel versions"
     We will use the config of the kernel that is currently running. If your running kernel is an older longterm/stable kernel, it's possible that some of the default choices for new options added to the kernel might not be what you want. You can replace `make olddefconfig` in the code block below with `make oldconfig` if you want to manually set new options. You can always later use `make menuconfig` to change kernel config options if you have issues.
 
+**for Arch Linux:** extract current config
 ```bash
 zcat /proc/config.gz > .config
+```
+**for Debian-based distros:** extract current config
+```bash
+sudo cp /boot/config-$(uname -r) .config
+```
+**for Debian-based distros:** or you might as well just [download the debian config](../attachments/debian.config) that was observed to have [suspend-workaround](../guides/postinstall.md) working on debian bookworm. Name the file as .config and place in current directory.
```

Comment:
Kernel configs change over time. You cannot make people use a single kernel config forever. Instead you need to find out what's special in this config that makes things work.

Reply:
removed in recent commit